src/main/java/eu/tornplayground/tornapi/models/faction/FactionAttacks.java

```diff
@@ -0,0 +1,170 @@
+package eu.tornplayground.tornapi.models.faction;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import eu.tornplayground.tornapi.models.Model;
+import eu.tornplayground.tornapi.models.converters.EpochLocalDateTimeConverter;
+import eu.tornplayground.tornapi.models.faction.attacks.Modifiers;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+public class FactionAttacks extends Model {
+    @JsonProperty("code")
+    private String code;
+
+    @JsonProperty("timestamp_started")
+    @JsonDeserialize(converter = EpochLocalDateTimeConverter.class)
+    private LocalDateTime timestampStarted;
+
+    @JsonProperty("timestamp_ended")
+    @JsonDeserialize(converter = EpochLocalDateTimeConverter.class)
+    private LocalDateTime timestampEnded;
+
+    @JsonProperty("attacker_id")
+    private Integer attackerId;
+
+    @JsonProperty("attacker_name")
+    private String attackerName;
+
+    @JsonProperty("attacker_faction")
+    private Integer attackerFaction;
+
+    @JsonProperty("attacker_factionname")
+    private String attackerFactionName;
+
+    @JsonProperty("defender_id")
+    private Integer defenderId;
+
+    @JsonProperty("defender_name")
+    private String defenderName;
+
+    @JsonProperty("defender_faction")
+    private Integer defenderFaction;
+
+    @JsonProperty("defender_factionname")
+    private String defenderFactionName;
+
+    @JsonProperty("result")
+    private String result;
```

Comment:
Since this is a limited field, I would say this could be an enum.

[Unofficial docs](https://tornapi.tornplayground.eu/faction/attacks#attack_result)

Reply:
added it to the class and the tests